highbury/chain.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "highbury",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.fury.black/",
+  "pretty_name": "Highbury",
+  "chain_id": "highbury_710-1",
+  "bech32_prefix": "fury",
+  "daemon_name": "fury",
+  "node_home": "$HOME/.fury",
+  "slip44": 459,
+  "alternative_slip44s": [
```

Comment:
is there an alternate, or was this copied from Kava, too?

Reply:
The alternate has now been updated to 710. Thanks for letting me know.
We intend to follow a similar approach as Irisnet.